README.md

```diff
@@ -1,2 +1,63 @@
 # Ansible Collection for GitHub
-The OHIO IT collection of Ansible modules for GitHub.
+
+### The OHIO IT collection of Ansible modules for GitHub.
+
+The collection includes a variety of Ansible modules to help automate the management of organizations, repositories, and user permissions in GitHub.
+
+---
+
+## Version Compatibility
+
+Tested against:
+`Ansible Version >=2.11`
+`Python >=3.8`
+`PyGitHub >=1.55`
+
+## Installation
+
+Install this collection via [Ansible Galaxy](https://galaxy.ansible.com/ohioit/github):
+
+```bash
+ansible-galaxy collection install ohioit.github
+```
+
+Content in this collection requires the [PyGitHub package](https://github.com/PyGithub/PyGithub) to interact with [GitHub REST API](https://docs.github.com/en/rest). You can install it with:
+
+```bash
+pip install PyGithub
+```
+
+## Included Content
+
+### Modules
+
+| Name                                                                                                                                             | Description                                                                            |
+| ------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------- |
+| [repository_info](https://github.com/senior-design-21-22/ansible-collection-github/blob/repo-information-module/docs/repository_information.rst) | Output repositories along with vital information from an (user specified) organization |
+
+## Usage
+
+### Repository Information (From organization)
+
+```
+  - name: "List GitHub repositories within an organization"
+      import_role:
+        name: ohioit.github.repository_info
+      vars:
+        github_token: "token"
+        org_name: "organization_name"
+```
```

Comment:
Add and example of how to use the module directly.

Reply:
This example has now been modified to work with the new collection format without roles.